GameData/RealismOverhaul/RO_SuggestedMods/ReStock/RO_Restock_RCSFix.cfg

```diff
@@ -0,0 +1,4 @@
+@PART[*]:HAS[@MODULE[ModuleRestockDepthMask]]:NEEDS[ReStock]:FOR[zzzRealismOverhaul]
+{
+	!MODULE[ModuleRestockDepthMask],*{}
```

Comment:
Is there a reason for this? It's a cool feature, why do we want to remove it?

Reply:
because it spams null refs

Reply:
Could you post one of these null refs?

Reply:
i would have to remove it and relaunch ksp, so give me a bit